app/components/match-history/match-history.html

```diff
@@ -1,12 +1,15 @@
 <div class="match-history-content container-fluid">
 	<div class="summoner-bar" ng-if="haveResults">
+
 		<div class="general-summoner-info">
 			<div class="summoner-name">Summoner: {{summoner.summonerName}}</div>
+			<h4 class="blue"><a ng-href="#/home/ranked-stats/{{summoner.summonerName}}">View ranked stats</h4></a>
```

Comment:
let's not use ng-href and instead and an ng-click event that will call a function on the controller's scope, which will handle the transition. Similar to the 'More stats for this match' button.